packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## v2.3.0 (Jan 19, 2024)
+## v2.3.1-preview.1 (Jan 23, 2024)
```

Comment:
Let's keep the existing "v2.3.0" heading (and not replacing it).

Reply:
Fixed